L1Trigger/TrackerDTC/python/ProducerES_cfi.py

```diff
@@ -3,11 +3,12 @@
 TrackTrigger_params = cms.PSet (
 
   # Parameter to check if configured Tracker Geometry is supported
+  # this refers to files included by Configuration/Geometry/python/GeometryExtended*_cff.py
   SupportedGeometry = cms.PSet (
     XMLLabel    = cms.string ("geomXMLFiles"                                    ), # label of ESProducer/ESSource
     XMLPath     = cms.string ("Geometry/TrackerCommonData/data/PhaseII/"        ), # compared path
     XMLFile     = cms.string ("tracker.xml"                                     ), # compared filen ame
-    XMLVersions = cms.vstring("TiltedTracker613", "TiltedTracker613_MB_2019_04", "OuterTracker616_2020_04", "OuterTracker800_2020_07" )  # list of supported versions
+    XMLVersions = cms.vstring("TiltedTracker613", "TiltedTracker613_MB_2019_04" )  # list of supported versions
```

Comment:
Either here, or in README.md, please say what one must do before updating this to a new geometry.